packages/create-atomic/templates/.env.example

```diff
@@ -1,33 +1,24 @@
 # The Plaform URL to use.
 # See https://docs.coveo.com/en/2976/coveo-solutions/deployment-regions-and-strategies
-ATOMIC_APP_PLATFORM_URL=https://platform.cloud.coveo.com
+PLATFORM_URL=https://platform.cloud.coveo.com
```

Comment:
Removing the ATOMIC_APP_ and copying the variables seemed like an extra, a bit useless, step?